### PR TITLE
Attempt to complete new city sign in challenge

### PR DESCRIPTION
**Pull request by alex-deaconu**
Add some logging
Clicking the "Enter city.." button redirects to the
old challenge page

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Verification page is here: screen shot 2017-05-03 at 2 50 59 pm

Logs of tests passing with New City Challenge: screen shot 2017-05-03 at 4 59 55 pm

@ezequielc @fjvallarino please review. Thanks!

**Comment by fjvallarino** [comment]
:+1:

File: lib/pages/googleAuthPage.js [hunk 1]
```diff
@@ -70,21 +79,30 @@ var GoogleAuthPage = function () {
...
+      });
+    });
+  };
+  
+  var _completeCityChallenge = function() {
+    return helper.wait(challengeCard, "Google Verify it's you page").then(function () {
+      console.log("Completing city challenge");
+
+      cityChallengeButton.click().then(function (){
+        var enter = "\ue007";
```
Comment (rodrigopavezi) [inline lib/pages/googleAuthPage.js]: Nice trick from @tejohnso :-)

Comment (alex-deaconu) [inline lib/pages/googleAuthPage.js]: @rodrigopavezi yup. Actually there's a practical application. I saw one instance where the submit button was covered by an autosuggest dropdown for the city.

Comment (rodrigopavezi) [inline lib/pages/googleAuthPage.js]: @alex-deaconu nice, good to know. cheers

**Pull request merged**